src/tink/http/Client.hx

```diff
@@ -32,7 +32,7 @@ interface ClientObject {
 }
 
 class StdClient implements ClientObject {
-  var worker:Worker;
+  var worker:Worker = Worker.EAGER;
```

Comment:
There was a time when instance variables could only be initialized with constant expressions and according to the manual that did not change. Good to know.